internal/primitive/transform/action/strings/capitalize_sentence.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2022 Linkall Inc.

```

Comment:
```suggestion
// Copyright 2023 Linkall Inc.

```

Reply:
thank  you for review !!! I changed copyright year.